pypackages/libsql-client/libsql_client/http_driver.py

```diff
@@ -8,18 +8,20 @@
 
 class _HttpDriver(_Driver):
     _session: aiohttp.ClientSession
+    _url: str
 
     def __init__(self, url: str) -> None:
-        self._session = aiohttp.ClientSession(url)
+        self._session = aiohttp.ClientSession()
+        self._url = url
 
     async def batch(self, stmts: List[_RawStmt]) -> List[ResultSet]:
         req_body = {
             "statements": [_encode_stmt(stmt) for stmt in stmts],
         }
 
-        async with await self._session.post("/", json=req_body) as resp:
+        async with await self._session.post(self._url, json=req_body) as resp:
             resp.raise_for_status()
-            resp_body = await resp.json()
+            resp_body = await resp.json(content_type=None)
```

Comment:
Because there are old databases already running that don't deploy the fix. Doug encountered the problem in Turso, so let's just keep the workaround for backward compatibility.

Reply:
https://github.com/chiselstrike/iku-turso-api/issues/122